client/src/commands.ts

```diff
@@ -86,7 +93,7 @@ export function reloadImportRegistries(
   _context: vscode.ExtensionContext,
   { client }: DenoExtensionContext,
 ): Callback {
-  return () => client.sendRequest(reloadImportRegistriesReq);
+  return () => client?.sendRequest(reloadImportRegistriesReq);
```

Comment:
Feels dodgy, why does TS think this is possibly undefined now?  Feels like we are missing something here.

Reply:
Ok, saw below the change in the types, it makes sense here, though are we sure we want to bind early like this?  Don't we want `extensionContext.client?.sendRequest(reloadImportRegistriesReq)` so we always use the currently assigned client?

Reply:
Good catch!